crates/core_arch/src/arm/neon/generated.rs

```diff
@@ -1941,6 +1941,42 @@ pub unsafe fn vcaleq_f32(a: float32x4_t, b: float32x4_t) -> uint32x4_t {
     vcageq_f32(b, a)
 }
 
+/// Floating-point convert to signed fixed-point, rounding toward zero
+#[inline]
+#[target_feature(enable = "neon")]
+#[cfg_attr(target_arch = "arm", target_feature(enable = "v7"))]
+#[cfg_attr(all(test, target_arch = "arm"), assert_instr(vcvt))]
+pub unsafe fn vcvt_s32_f32(a: float32x2_t) -> int32x2_t {
```

Comment:
It doesn't seem like there's any assert_instr for aarch64 for these?!

Reply:
There are different implementations for arm and aarch64. You can use https://godbolt.org/ to verify

Reply:
Well it's fcvtzs and co., so idk why those aren't being asserted for here.

Reply:
https://godbolt.org/z/GTrMP6 In this version of the test, the arm uses the fptosi command. The test link comes from the top link [here](https://github.com/rust-lang/stdarch/issues/148)

Reply:
The arm version does not need to introduce external links #[link_name = ...]. which aarch64 needs

Reply:
I think you are right, thanks a lot